internal/installercache/installercache_test.go

```diff
@@ -0,0 +1,71 @@
+package installercache
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("installer cache", func() {
+	var (
+		cache      *ReleaseCache
+		r1, r2, r3 string
+	)
+
+	BeforeEach(func() {
+		cache = New(2, logrus.New())
+	})
+
+	AfterEach(func() {
+		_ = os.Remove(r1)
+		_ = os.Remove(r2)
+		_ = os.Remove(r3)
+	})
+
+	addFile := func(releaseID string) *release {
+		cache.releases.ContainsOrAdd(releaseID, &release{})
+		r, _ := cache.releases.Get(releaseID)
```

Comment:
```suggestion
		r, err := cache.releases.Get(releaseID)
		Expect(err).ShouldNot(HaveOccurred())
```

to make sure we're not missing the more immediate error-paths in case something goes wrong there

Reply:
The other value is not error but ok (flag that indicates whether the value exists). In our case it guaranteed to exist.